picoquictest/mediatest.c

```diff
@@ -1190,7 +1301,25 @@ int mediatest_video_data_audio_test()
     spec.do_video = 1;
     spec.do_audio = 1;
     spec.data_size = 10000000;
-    ret = mediatest_one(3, &spec);
+    ret = mediatest_one(mediatest_video_data_audio, &spec);
+
+    return ret;
+}
+
+int mediatest_video2_down_test()
+{
+    int ret;
+    mediatest_spec_t spec = { 0 };
+    spec.ccalgo = picoquic_bbr_algorithm;
+    spec.bandwidth = 0.01;
+    spec.do_video = 1;
+    spec.do_video2 = 1;
+    spec.do_audio = 1;
+    spec.data_size = 0;
+    spec.latency_average = 200000;
```

Comment:
This is a new test added recently to picoquic master branch. This test fails often via GitHub workflows/runners. It fails about 1 out of 15 times on MacOS.  The issue appears to be with latency average and max.  Adjusting these resolves the test failures. Need to upstream this issue to picoquic/master.